samples/TestServerWithHosting/TestServerWithHosting.csproj

```diff
@@ -5,6 +5,10 @@
     <TargetFramework>net9.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <!--
+    Serilog AOT support regressed
+    <PublishAot>true</PublishAot>
```

Comment:
FYI I'm going to fix this on the serilog side, I just didn't want to block this PR.

Reply:
How did serilog regress?

Reply:
I had to use a feature switch in their code, and this was before our feature-switch-attribute system, so it was very delicate. Looks like a refactoring broke it and they didn't re-run the AOT-test-binary.